index.html

```diff
@@ -363,10 +361,9 @@ <h4 class="media-heading">No More Operations</h4>
 	    <div class="media-body">
 	      <h4 class="media-heading">Make Money</h4>
 	      <p>
-		Sell your application on our app store,
-		or sell virtual goods with in-app purchases.
-		Instant deployment at scale makes it trivial for
-		newcomers to challenge goliaths in the industry.
+		Take donations, show advertisements,
+		or charge for real world services.
+    There are plenty of business models that don't betray user trust.

```

Comment:
There's an element of, sell access to your software, or components of software (in-app purchases). Do you agree?